fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -48,7 +48,7 @@ public class IcebergTable extends Table {
 
     public static final String ICEBERG_CATALOG = "starrocks.catalog-type";
```

Comment:
why not "iceberg.catalog.type"?
it is wired to use starrocks as beginning.

Reply:
I'll submit a separated PR to refractor this property after this one is merged.